basetest.pm

```diff
@@ -325,15 +328,26 @@ sub run_post_fail {
         # Read them now to not stumble upon them in the next module.
         $self->get_new_serial_output();
     }
+
     $self->fail_if_running();
+    $self->compute_test_execution_time();
+    my $post_fail_hook_execution_time = execution_time($post_fail_hook_start_time);
+    bmwqemu::diag(sprintf("||| post fail hooks runtime: %d s", $post_fail_hook_execution_time));
```

Comment:
Would you mind adding a unit test for that?

Reply:
Done, now we look for the specific pfh message.